lms/templates/courseware/progress.html

```diff
@@ -80,7 +80,10 @@ <h2 class="title">${_("We're working on it...")}</h2>
                     <p class="copy">${_("We're creating your certificate. You can keep working in your courses and a link to it will appear here and on your Dashboard when it is ready.")}</p>
                   </div>
                   <div class="msg-actions"></div>
-                  %elif not is_id_verified or is_unverified:
+                  %elif missing_required_verification or is_unverified:
```

Comment:
What if the student isn't missing required verification but is unverified according to https://github.com/edx/edx-platform/blob/e6137dd6af7a6df9a7f92a2975fd47f73c6f9c3b/lms/djangoapps/certificates/api.py#L207-L238?